test/mean-module.test.js

```diff
@@ -0,0 +1,65 @@
+'use strict';
+
+var path = require('path'),
+  helpers = require('yeoman-generator').test,
+  assert = require('yeoman-generator').assert,
+  temp = require('temp').track();
+
+describe('Module Sub Generators Tests', function () {
+  this.timeout(0);
+  /**
+   * Setup the temp directory
+   */
+  before(function (done) {
+    helpers.testDirectory(path.join(__dirname, 'temp'), done);
+  });
+
+  /**
+   * Clean up temp directory
+   */
+  after(function () {
+    temp.cleanup();
+  });
+
+  describe('Generate an express controller through the sub-generator', function () {
+    beforeEach(function (done) {
+      helpers.run(path.join(__dirname, '../mean-module'))
+        .withOptions({
+          'skip-install': true
+        })
+        .withArguments(['foo'])
```

Comment:
@ilanbiala Changed name to prompt on all of them. Please, check it out when you have time.

I will re-check for forgotten line comments too.